trio_async/worker.py

```diff
@@ -0,0 +1,66 @@
+import asyncio
+import logging
+import os
+import sys
+
+import trio_asyncio
+from temporalio.client import Client
+from temporalio.worker import Worker
+
+from trio_async import activities, workflows
+
+
+@trio_asyncio.aio_as_trio  # Note this decorator which allows asyncio primitives
```

Comment:
As above, would be good to expand and clarify this. Are we saying

> without this decorator you wouldn't be able to use anything from the `asyncio` module.

?

Reply:
Yes, I can adjust. I do not believe it is true that it's _anything_ from the asyncio module because I didn't go over everything in that module.